packages/main/src/components/emoji/utils/Api.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const endpoint = "https://api.zuri.chat";
+const channelsEndpointV1 = "https://channels.zuri.chat/api/v1";
```

Comment:
- these are constants you should import from the zuri utilities
- you aren't using the channels endpoint